### PR TITLE
Use lite pressed fronts in Facia

### PR DESCRIPTION
**Pull request by QuarpT**
## What does this change?

Use the lite fronts pressed by facia-press in Facia.

Can merge this to master after https://github.com/guardian/frontend/pull/18364 is merged

## What is the value of this and can you measure success?

Facia can process smaller Json files for most requests, more performant - we be able to handle a higher request rate.

## Tested in CODE?

I will

### CODE REVIEW

File: facia/app/controllers/FaciaController.scala [hunk 1]
```diff
@@ -218,13 +218,13 @@ trait FaciaController extends BaseController with Logging with ImplicitControlle
 
   private def getPressedCollection(collectionId: String): Future[Option[PressedCollection]] =
```
Comment (TBonnin) [inline facia/app/controllers/FaciaController.scala]: Are we sure this function is not used by endpoints that requires the full pressed file? Same question below for `getSomeCollections`?

**Review by TBonnin — APPROVED** [review]
Looks good to me. I just have one question.
Let's test this thoroughly in CODE.

File: facia/app/controllers/FaciaController.scala (hunk 1 above)
Comment (QuarpT) [reply facia/app/controllers/FaciaController.scala]: I'm not certain. We need to list the endpoints require the full PressedPage.

For this PR, I've assumed that only `/*path/show-more/*id.json`and `/*path/deduped.json` require the full version.

Comment (TBonnin) [reply facia/app/controllers/FaciaController.scala]: let's try to figure it out. Let me know if you need any help.

File: facia-press/app/frontpress/FapiFrontPress.scala [hunk 2]
```diff
@@ -148,29 +150,61 @@ trait FapiFrontPress extends Logging {
...
-      val trimmedBackfill = backfill.take(maxStories - trimmedCurated.length)
-      PressedCollection.fromCollectionWithCuratedAndBackfill(
-        collection,
-        trimmedCurated,
-        trimmedBackfill,
-        treats
+        .toOption(storyCountTotal)
+        .getOrElse(Math.min(Configuration.facia.collectionCap, storyCountTotal))
+      val storyCountVisible = Container.storiesCount(CollectionConfig.make(collection.collectionConfig), curated ++ backfill).getOrElse(storyCountMax)
+      val hasMore = storyCountVisible < storyCountMax
```
Comment (QuarpT) [inline facia-press/app/frontpress/FapiFrontPress.scala]: storyCountVisible should probably be min(storyCountVisible, storyCountMax)

Comment (TBonnin) [reply facia-press/app/frontpress/FapiFrontPress.scala]: `storyCountVisible` is either the number of stories necessary to fill the expected layout or `storyCountMax`.
In theory no container layout requires more than the max cap

**Comment by TBonnin** [comment]
Time spent by Facia to read json
screen shot 2017-12-06 at 17 26 38

Facia CPU
screen shot 2017-12-06 at 17 27 09

I'll let you guess what time we deployed this change. 😁

**Comment by TBonnin** [comment]
And the median response time for Facia (excluding the internal redirects)
screen shot 2017-12-07 at 10 42 37

**Pull request merged**